tmd/Topology/vectorizations.py

```diff
@@ -0,0 +1,112 @@
+"""Topology vectorization algorithms."""
+
+# Copyright (C) 2022  Blue Brain Project, EPFL
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# pylint: disable=invalid-slice-index
+import copy
+
+import numpy as np
+from scipy import stats
+
+from .statistics import get_lengths
+
+
+def get_limits(phs_list):
+    """Returns the x-y coordinates limits (min, max) for a list of persistence diagrams."""
+    ph = copy.deepcopy(phs_list)
```

Comment:
Why this copy? `np.transpose` does not change the input array.

Reply:
I think you are right, I kept it just in case, but I don't think it's needed.

Reply:
ok